src/parsers/power_models_data.jl

```diff
@@ -900,41 +902,146 @@ function read_3w_transformer!(
     end
 end
 
-function make_dcline(name::String, d::Dict, bus_f::ACBus, bus_t::ACBus)
-    return TwoTerminalHVDCLine(;
+function make_dcline(name::String, d::Dict, bus_f::ACBus, bus_t::ACBus, source_type::String)
+    if source_type == "pti"
```

Comment:
Could use `Val` to dispatch on `source_type`

Reply:
It's possible but an overkill IMO, so we should be fine